docs/1-Getting-Started/2-Starting-new-project.md

```diff
@@ -1,12 +1,6 @@
-> We use Laravel as an example, but **kool** is a **multi-stack** tool that can be used with any language and framework. [Check out our presets](/docs/presets/introduction) for easily creating projects using other frameworks (e.g. WordPress, Symfony, Next.js, Nuxt.js, NestJS, Adonis, Hugo, etc).
+> We use PHP for our Hello World example, but **kool** is a **multi-stack** tool that can be used with any language and framework. For instance, [check out our presets](/docs/presets/introduction) for easily creating projects using JavaScript frameworks like Next.js, Nuxt.js, NestJS, and AdonisJs.
```

Comment:
Also: what about instead of

> for easily creating projects using JavaScript frameworks like Next.js, Nuxt.js, NestJS, and AdonisJs.

we go simply:

> for easily creating projects using other stacks (like JavaScript).

I mean, it is a listing, that is not realistic we will keep adding to that list as we add more presets, so it just feels incomplete.

Reply:
@fabriciojs - changed "multi-stack" to "stack agnostic". Did _not_ change the "for easily creating projects" sentence.